ross/shaft_element.py

```diff
@@ -142,6 +142,10 @@ def __init__(
         self.Ie = np.pi * (o_d ** 4 - i_d ** 4) / 64
         phi = 0
 
+        # Slenderness ratio of beam elements (G*A*L^2) / (E*I)
+        sld = (self.material.G_s * self.A * self.L ** 2) / (self.material.E * self.Ie)
+        self.sld_ratio = sld        
```

Comment:
I think `self.slenderness_ratio = sld` will be more clear to the user.

Reply:
Talking about slenderness ratio (SR), I had an idea to better warn the user if some shaft element has it small enough to might be a problem: plotting these elements in a different color, like a "visual warning". While the elements that are OK are plotted in gray, the elements with SR smaller than 30 are plotted in yellow. Take a look in these plots here: 

Here, all the elements have their SR above 30.
![rotor1](https://user-images.githubusercontent.com/45969994/57990591-7fcb2b00-7a7e-11e9-9a07-8e2c15ab54a1.PNG)

and here, some elements have their SR below 30.
![rotor2](https://user-images.githubusercontent.com/45969994/57990622-bacd5e80-7a7e-11e9-9667-32072f85687e.PNG)

What do you think about it?